assets/css/screen-detail.scss

```diff
@@ -3,6 +3,34 @@
   border-radius: 8px;
   padding: 24px 16px;
 
+  &--paess-g {
```

Comment:
Colors look good, but there seems to be some vertical alignment issues. In the designs, the location "header" is a bit slimmer than what is currently being shown, and the `margin-top` is designed to be less than other screens. The height can be solved by changing `70px` to `40px` in the below `linear-gradient` functions (is that what they are called in CSS?). The margins may take a little more work. I think I have the needed changes but it's across multiple files so the suggestion feature won't work. Just reach out on Slack when you get time and I'll show you what I have.

Reply:
Had to do a little bit extra to get around the padding/margin stuff. Basically, removing the padding caused other simulations to end up more flushed to the bottom of their containers so instead, I just added some PAESS specific overrides of the padding values and then set the margins to the right values below in `screen-detail__screen-type-location--paess`. That seems to have worked in keeping the rest of the screen previews the same while letting us adjust the PAESS header to the way we need it